resources/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Identifying CPD Officers Who Have Killed
+header: no
+#permalink: /media/
+mediaplayer: false
+---
+
+## Background
+
+Due to language in the Chicago Police Department's [contract](https://www.dnainfo.com/chicago/20160106/downtown/inside-chicago-police-union-contract-document-shows-rules-of-force) officers who kill on the job are shielded from public disclosure, which is not the case in other major cities like Oakland.  Since City Hall and CPD won't identify killer officers, this guide should serve as a skeleton resources for victim's families, journalists, and citizens and others to identify the officers. Here are some basic steps you can take when a CPD officer shoots and kill someone, and how to gather information. This guide is loosely based on the [Chicago Reader](http://www.chicagoreader.com/Bleader/archives/2016/01/14/internal-police-records-point-to-the-identity-of-the-officer-who-fatally-shot-bettie-jones-and-quintonio-legrier) article confirming open source research that Lucy Parsons Labs was involved in. 
```

Comment:
s/skeleton resources/skeleton resource/;
", and citizens and others" is just weird now. maybe drop the "and citizens"?